src/main/java/org/entando/kubernetes/security/AuthorizationFilter.java

```diff
@@ -0,0 +1,126 @@
+package org.entando.kubernetes.security;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.regex.Pattern;
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpFilter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.entando.kubernetes.model.web.response.SimpleRestResponse;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+@Component
+public class AuthorizationFilter extends HttpFilter {
+
+    public static final Pattern SERVLET_PATH_REGEX = Pattern.compile("^((/components)|(/registries)).*");
+    protected static final List<String> ECR_PERMISSION_LIST = Arrays.asList("superuser", "enterECR");
+    protected static final List<String> ALLOWED_METHODS = Arrays.asList(HttpMethod.PUT.name(), HttpMethod.POST.name(),
```

Comment:
why not GET as well? Do we really need to match specific methods?

Reply:
We evaluated that endpoint fetching data without writing anything can be outside of this process in order to avoid overloading the CM with a lot of requests. The major case is the polling during the installation. Anyway we can decide to change this behaviour

Reply:
Anyway we decided to remove the check on the HTTP method and to invert the approach by allowing only those 3 endpoints used to poll against a bundle installation/uninstallation operation